pkgs/racket-doc/scribblings/reference/hashes.scrbl

```diff
@@ -212,6 +212,16 @@ mappings can hide earlier mappings.
 
 See also @racket[make-custom-hash].
 
+@examples[
+#:eval the-eval
+(make-hash)
+(make-hash '([0 . 1] [1 . 2] [2 . 3]))
+(make-hash '([0 . 1] [1 . 2] [0 . 3]))
+(make-hash (list (cons 0 1) (cons 1 2) (cons 2 3)))
+(make-hash '((0 1) (1 2) (2 3)))
+(make-hash (list (cons "hello" "world") (cons 42 "meaning of life") (cons (list 1 2 3) (list #\a #\b #\c)) (cons #t #f) (cons 'apple 'orange) (cons (list 'a 1 "apple") (vector #t 'b 2 "banana")) (cons + -)))
```

Comment:
I think this is a bit too much, and it wouldn't be pretty in the rendered doc.

<img width="1261" alt="Screenshot 2024-02-02 at 7 25 13 AM" src="https://github.com/racket/racket/assets/9099577/6ba080c2-6d87-46a6-88f8-52e9aa125a5f">

Reply:
haha, yes that is _very_ reasonable. I will trim it back